.env

```diff
@@ -0,0 +1 @@
+REACT_APP_JWT_SECRET=himadri
```

Comment:
You can add the .env file your .gitignore file so it will not get uploaded on github.

Reply:
Yes, thank you for pointing it out.